src/main/resources/org/jenkinsci/plugins/GithubSecurityRealm/config.jelly

```diff
@@ -16,5 +16,8 @@
 		<f:entry title="Client Secret" field="clientSecret" help="/plugin/github-oauth/help/realm/client-secret-help.html">
 			<f:textbox />
 		</f:entry>
+		<f:entry title="OAuth Scope(s)" field="oauthScopes" help="/plugin/github-oauth/help/realm/oauth-scopes-help.html">
+			<f:textbox default="repo,read:org" />
```

Comment:
It's not as much about borking the server as being easier to use (not requiring I go read the GitHub API docs) and being easier to debug.